From efea7d243ac86acc4b4e8a1354dd22de95afe112 Mon Sep 17 00:00:00 2001
From: Erez Shaul <eshaul@habana.ai>
Date: Tue, 27 Aug 2024 18:23:22 +0300
Subject: [PATCH] [SW-198620] fs1-palad: do not call spi_flash_prot_early

do not call spi_flash_prot_early in fs1-palad
it is redundant and can halt the run in palladium

Change-Id: Ibb69e4aac9dcab4414885c700fee1b95afdf23ae
Signed-off-by: Erez Shaul <eshaul@habana.ai>
---

diff --git a/arch/fs1/arc/palad.c b/arch/fs1/arc/palad.c
index 637d369..0b47494 100644
--- a/arch/fs1/arc/palad.c
+++ b/arch/fs1/arc/palad.c
@@ -292,3 +292,7 @@
 	preboot_puts("palad-bloader resigned\n");
 	image_start(APP_RST_VECTOR_ADDR);
 }
+
+void spi_flash_prot_early(void)
+{
+}
